src/Stratis.Bitcoin.Cli/Program.cs

```diff
@@ -10,53 +14,136 @@ namespace Stratis.Bitcoin.Cli
     class Program
     {
         /// <summary>
-        /// The expected sequence of arguments is [network-name] [rpc-command] [rpc-params].
+        /// The expected sequence of arguments:
+        ///    1) [network-name] [rpc-command] [rpc-params].
+        /// OR
+        ///    2) [network-name] [api-controller "/" api-command] [api-params].
         /// </summary>
         public static void Main(string[] args)
         {
             try
             {
-                if (NodeSettings.PrintHelp(args, Network.Main))
+                // Display help if required.
+                if (args.Length == 0 || args.Contains("-help") || args.Contains("--help"))
                 {
-                    // TODO: add more outputs
-                    // Proposal: use reflection to find all settings classes and print out help.
-                    RpcSettings.PrintHelp(Network.Main);
+                    var builder = new StringBuilder();
+                    builder.AppendLine("Usage:");
+                    builder.AppendLine(" dotnet run <Stratis.Bitcoin.Cli/Stratis.Bitcoin.Cli.dll> [network-name] [command] [arguments]");
+                    builder.AppendLine();
+                    builder.AppendLine("Command line arguments:");
+                    builder.AppendLine();
+                    builder.AppendLine($"[network-name]            Name of the network - e.g. \"stratis\".");
+                    builder.AppendLine($"[command]                 Name of RPC method or API <controller>/<method>.");
+                    builder.AppendLine($"[arguments]               Argument by position (RPC) or Name = Value pairs (API).");
```

Comment:
why the three lines with interpolated strings here?

and why to use string builder at all here and not just Console.WriteLine line by line?

if you cared about writing all at once (which i don't think you do), you could do it like this anyway:
```
Console.WrilteLine("Usage:\n"
    + "second line ...\n"
    + "third line line ...\n"
    + "last line");
```

Reply:
I've removed the '$' signs.

Consistency first. The purpose of the StringBuilder is to remain consistent with the code found in the PrintHelp methods found in "RpcSettings.cs" and "MinerSettings.cs". If we are going to change it we should do it in those methods as well. However that  is not the aim of this PR.

Reply:
I'd rather use the StringBuilder here as it's easier to write than to use \n.

Reply:
@quantumagi sure, no problem, i've just notice it here first, so just random comment, feel free to skip as it is off-topic

@bokobza then i think direct Console.WriteLine is better